.spectral.yaml

```diff
@@ -58,7 +58,7 @@ rules:
   schema-examples:
     description: Schemas must have non-empty "x-examples" object.
     severity: error
-    given: $.components.schemas.*
+    given: $.components.schemas[*]
```

Comment:
Making the JSON path syntax consistent with Spectral's use of the syntax:
https://github.com/stoplightio/spectral/blob/develop/packages/rulesets/src/oas/index.ts#L338